Saturation/SaturationAlgorithm.cpp

```diff
@@ -1387,6 +1414,8 @@ MainLoopResult SaturationAlgorithm::runImpl()
       if (env.timeLimitReached()) {
         throw TimeLimitExceededException();
       }
+
+      env.statistics->activations = l;
```

Comment:
Huh? `+= 1`?

Reply:
You mean MOV vs INC? Is it a big difference?

Reply:
Oh, now I see. I read that as `= 1`, not `= l`, but there is an `l` in scope. Still - if we throw an exception for e.g. timeout, do we not want to report the number of activations? But no big deal, I've got it now.